cmd/testscript/help.go

```diff
@@ -14,7 +14,7 @@ The testscript command runs github.com/rogpeppe/go-internal/testscript scripts
 in a fresh temporary work directory tree.
 
 Usage:
-    testscript [-v] [-e VAR]... files...
+    testscript [-v] [-e VAR]... [-u] files...
```

Comment:
Please update `README.md` too, as it holds a copy of this message :\

Reply:
Ugh. It really would be nice not to have to worry about this. Thanks for the catch!